packages/web-components/src/theme/set-theme.stories.ts

```diff
@@ -1,17 +1,38 @@
-import { html } from '@microsoft/fast-element';
+import { html, repeat, when } from '@microsoft/fast-element';
 import { teamsDarkTheme, teamsLightTheme, webDarkTheme, webLightTheme } from '@fluentui/tokens';
-
-import type { Story } from '../helpers.stories.js';
+import type { Meta } from '../helpers.stories.js';
 import { renderComponent } from '../helpers.stories.js';
 import type { Theme } from './set-theme.js';
 import { setTheme } from './set-theme.js';
+import { colorNeutralBackground2, colorNeutralForeground2 } from './design-tokens.js';
+
+const themes: Map<string, Theme | null> = new Map([
+  ['web-light', webLightTheme],
+  ['web-dark', webDarkTheme],
+  ['team-light', teamsLightTheme],
+  ['team-dark', teamsDarkTheme],
+  ['unset', null],
+]);
+
+function updateTheme(evt: Event, type = 'global') {
+  const { value } = evt.target as HTMLSelectElement;
 
-const fluentTheme: Record<string, Theme> = {
-  webLightTheme,
-  webDarkTheme,
-  teamsLightTheme,
-  teamsDarkTheme,
-};
+  if (themes.has(value) !== undefined) {
+    switch (type) {
+      case 'global':
+        setTheme(themes.get(value)!);
+        break;
+      case 'local':
+        setTheme(themes.get(value)!, document.querySelector('.local') as HTMLElement);
+        break;
+      case 'shadow':
+        document.querySelectorAll('.shadow').forEach(el => {
+          setTheme(themes.get(value)!, el as HTMLElement);
+        });
+        break;
+    }
+  }
+}
```

Comment:
Is this our custom theme switcher? Or is there an opportunity to use Storybook's built-in toolbar? (Coudl be a different ticket) 

https://storybook.js.org/docs/essentials/toolbars-and-globals

Reply:
This is specific for this story. This is what the story looks like. There are 3 `<select>`s to set themes for global, a light DOM element, and an element with shadow DOM. Also, I think it's better to make the theme settings as part of the story instead of being part of the Storybook UI for clarity.

<img width="996" alt="image" src="https://github.com/user-attachments/assets/35176560-aeed-4cce-83d6-14f242939aba">